<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialisationQtClass</class>
 <widget class="QMainWindow" name="InitialisationQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>686</width>
    <height>473</height>
=======
    <width>481</width>
    <height>400</height>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
   </rect>
  </property>
  <property name="windowTitle">
   <string>InitialisationQt</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>30</x>
      <y>160</y>
      <width>181</width>
=======
      <x>310</x>
      <y>260</y>
      <width>121</width>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
      <height>31</height>
     </rect>
    </property>
    <property name="text">
<<<<<<< HEAD
     <string>Afficher un message</string>
=======
     <string>Afficher Message</string>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>10</x>
      <y>210</y>
=======
      <x>110</x>
      <y>140</y>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
<<<<<<< HEAD
      <width>181</width>
=======
      <width>121</width>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
      <height>31</height>
     </rect>
    </property>
    <property name="text">
<<<<<<< HEAD
     <string>Connexion au server hercules</string>
=======
     <string>Connexion</string>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>220</x>
      <y>10</y>
      <width>201</width>
      <height>16</height>
=======
      <x>240</x>
      <y>30</y>
      <width>201</width>
      <height>61</height>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
     </rect>
    </property>
    <property name="text">
     <string>Status connexion : Déconnecté</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
<<<<<<< HEAD
      <x>30</x>
      <y>120</y>
      <width>181</width>
=======
      <x>64</x>
      <y>220</y>
      <width>111</width>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
      <height>31</height>
     </rect>
    </property>
    <property name="text">
<<<<<<< HEAD
     <string>Envoyer un message au server</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Status connexion : Message recu :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ClientStatusLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat client :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ClientStatusLabel_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Status connexion : Message recu :</string>
=======
     <string>Envoyer au server</string>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< HEAD
     <width>686</width>
=======
     <width>481</width>
>>>>>>> eb7c2e3668728ec3c0fb885685ec9d92fdbdfa85
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="InitialisationQt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>InitialisationQtClass</receiver>
   <slot>Connexion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>InitialisationQtClass</receiver>
   <slot>onDisplayMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>InitialisationQtClass</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onDisplayMessageButtonClicked()</slot>
  <slot>Connexion()</slot>
  <slot>onSendMessageButtonClicked()</slot>
 </slots>
</ui>
